### PR TITLE
Added "video" and "partner bio" to curriculum overview table of contents

### PR DESCRIPTION
**Pull request by orangemug**
## Description
Added "video" and "partner bio" to curriculum overview table of contents

Screenshot 2024-10-07 at 11 41 19

## Issue(s)

Fixes `CUR-939`

## How to test
Cycle 1

1. Go to https://deploy-preview-2855--oak-web-application.netlify.thenational.academy
2. Check for regressions

Cycle 2

1. Checkout locally with `ENABLE_CYCLE_2=true`
2. Check "Video Guide" and "Our curriculum partner" work as expected in all viewport sizes

### CODE REVIEW

**Review by sonalisin — APPROVED** [review]

File: src/components/CurriculumComponents/OverviewTab/OverviewTab.tsx [hunk 1]
```diff
@@ -186,20 +186,36 @@ const OverviewTab: FC<OverviewTabProps> = (props: OverviewTabProps) => {
...
   const h1Headings = (curriculumExplainer.explainerRaw ?? []).filter(
     (block) => {
       return block.style === "heading1";
     },
   );
 
+  if (isVideoEnabled) {
+    h1Headings.push({
+      _key: "video-guide",
+      children: [{ text: "Video Guide" }],
```
Comment (rightsaidjames) [inline src/components/CurriculumComponents/OverviewTab/OverviewTab.tsx:204]: Sentence case!
```suggestion
      children: [{ text: "Video guide" }],
```

**Review by rightsaidjames — CHANGES_REQUESTED** [review]
In addition to the below copy change, I've noticed that the table of contents buttons (that will eventually be links) don't have focus styles, which is a major a11y violation. Happy for this to be fixed in another ticket, but I'd strongly prefer we don't go live with it.

**Review by rightsaidjames — APPROVED** [review]
Looks good - the TOC now behaves identically to the 'plan a lesson' page table of contents, with fully accessible links (rather than buttons). The Video guide (where relevant) and Our curriculum partners sections are present in the list and take the user to the right part of the page in all viewports.

**Review by rightsaidjames — APPROVED** [review]
Retested alongside #2865 after rebase - LGTM

**Pull request merged**